### ISSUE TITLE
/gcmd?

### ISSUE THREAD

**Issue by rovertdude** [TotalFreedom/TotalFreedomMod#1651]
Apparently Rcon was recently breached or someone went rogue - somehow, Rcon was able to /gcmd or similar adamvandyck to /doom admins and more.
Any ideas on how they did it?
We found that Rcon did only once /gc which may have been /gcmd

**Comment by Wild1145** [comment]
RCON cannot GCMD as I recall, and I dont know what gc is...

**Comment by RobinGall2910** [comment]
/gc = the /lag command in Essentials

**Comment by Wild1145** [comment]
Then I dont see how it could have forced anyone to doom anyone... It couldnt gcmd last time I remember...

**Comment by rovertdude** [comment]
@Wild1145 Somehow, Rcon managed to use a command and force him to use /doom.
I have absolutely no clue how they did it.

**Comment by PacksGamingHD** [comment]
Rcon somehow made adam doom

I'm pretty sure there was a bug where if a person was logged into telnet they could to telnet commands in game

**Comment by Wild1145** [comment]
@rovertdude I just checked. RCON cannot gcmd other players, nor can it doom them...

**Comment by JeromSar** [comment]
@Wild1145 This makes sense, RCON has been [severely limited](https://github.com/TotalFreedom/TotalFreedomMod/commit/cdf2dc676018f52aa9c093258502b0b0c42f722f) lately.

**Comment by Wild1145** [comment]
@JeromSar The issue is that its not RCON thats doing this... Its the players themselves through signs... And I cannot figure out how...

**Comment by Wild1145** [comment]
Superseded by #1831
